### PR TITLE
Added downsample tests

### PR DESCRIPTION
**Pull request by pgdr**
I have now added tests for the `downsample` function.

Thanks, @joakim-hove for help!

### CODE REVIEW

**Comment by pgdr** [comment]
@joakim-hove Still the same error:
```python
ImportError: Failed to import test module: friskby.tests.test_quick_view
Traceback (most recent call last):
  File "/usr/lib/python2.7/unittest/loader.py", line 254, in _find_tests
    module = self._get_module_from_name(name)
  File "/usr/lib/python2.7/unittest/loader.py", line 232, in _get_module_from_name
    __import__(name)
  File "/home/pgd/friskby/friskby/friskby/tests/test_quick_view.py", line 8, in <module>
    from sensors.views.quick import downsample
ImportError: No module named sensors.views.quick
```

**Comment by pgdr** [comment]
WIP in progress.

File: friskby/tests/test_quick_view.py [hunk 1]
```diff
@@ -5,11 +5,29 @@
 
 from .context import TestContext
 from sensor.models import *
+from views.quick import downsample
```
Comment (joakim-hove) [inline friskby/tests/test_quick_view.py]: `from friskby.quick.views import downsample`

Comment (pgdr) [inline friskby/tests/test_quick_view.py]: thanks! :)  that worked.  final commit imminent

**Pull request merged**